releasenotes/notes/1.3/add-gates-to-collect-clifford-af88dd8f7a2a4bf9.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 features_transpiler:
   - |
-    Add an argument ``matrix_based`` to the :class:`.CollectCliffords()` transpiler pass.
+    Add argument ``matrix_based`` to the :class:`.CollectCliffords()` transpiler pass.
```

Comment:
Just a comment for consistency: I think we mostly write in past-tense, right? I.e. `Fixed something` or `Added something` in case you'd like to change this 🙂